examples/tutorials/Part 0 - Installation Guide.md

```diff
@@ -0,0 +1,228 @@
+
+# Part 0: Installing PySyft
+
+Before you start following the tutorials, you need to install PySyft on your computer.
+
+Here are the PySyft installation instructions for Linux, MacOs and Windows.
+
+Just scroll down to find your preferred OS:
+
+--------------
+
+Authors:
+
+- Alan Aboudib - Twitter: [@alan_aboudib](https://twitter.com/alan_aboudib)
+
+
+
+----------------------------
+## 0.1 Linux
+--------------------
+
+Installing PySyft on Linux is really straight forward. Here are the steps:
+
+#### 1. Make sure you have python >= 3.6
+
+You can check out the version number by running:
+
+<font color='red'>$</font>  `python --version`
```

Comment:
You might want to put a link here for instructions on upgrading you python version. It can be a pain but this doc should not be a python install doc :slightly_smiling_face:

Reply:
I totally agree